cmd/server/inventory.go

```diff
@@ -96,7 +96,7 @@ func listBuilders() error {
 
 func printConfigValidator(v pkgadapter.ConfigValidator) {
 
-	fmt.Printf("Params: \n")
+	fmt.Print("Params: \n")
```

Comment:
The two front-ends(mixc and mixs) use normal stdout stuff. The core mixer uses glog.